tests/notifier/notifier_test.py

```diff
@@ -44,3 +172,62 @@ def test_can_convert_created_at_datetime_to_timestamp_string(self):
 
         self.assertEquals(expected_timestamps,
                           converted_timestamps)
+
+    @mock.patch(
+        'google.cloud.forseti.notifier.notifier.find_notifiers', autospec=True)
+    @mock.patch(
+        'google.cloud.forseti.notifier.notifier.scanner_dao', autospec=True)
+    def test_no_notifications_for_empty_violations(
+        self, mock_dao, mock_find_notifiers):
+        """No notifiers are instantiated/run if there are no violations.
+
+        Setup:
+            Mock the scanner_dao and make its map_by_resource() function return
+            an empty violations map
+
+        Expected outcome:
+            The local find_notifiers() function is never called -> no notifiers
+            are looked up, istantiated or run."""
+        mock_dao.map_by_resource.return_value = dict()
+        mock_srvc_cfg = mock.MagicMock()
+        mock_srvc_cfg.get_global_config.return_value = FAKE_GLOBAL_CONFIGS
+        mock_srvc_cfg.get_notifier_config.return_value = FAKE_NOTIFIER_CONFIGS
+        notifier.run('iid-1-2-3', mock.MagicMock(), mock_srvc_cfg)
```

Comment:
nit: `srvc` is a non-standard abbreviation, can you please change this to `mock_service_config` to match the code in the app, and be consistent?
https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/google/cloud/forseti/notifier/notifier.py#L122

Please change this everywhere.

Reply:
Done.